TG-Netflix/src/pages/my-list/MyList.jsx

```diff
@@ -75,14 +74,15 @@ export default function MyList() {
               data={{ coords: coords, dataset: dataset }}
               onClick={openBigModal}
               movieID={movieID}
+              
             />
             
           )}
```

Comment:
Verplaats dit hele blok (69 t/m 80) buiten de GridContainer (thans GridLayout), bv onder TitleWrapper. Dit lost dat rare styling-probleem op in de kleine modal wat we vorige week zagen.